app/src/main/AndroidManifest.xml

```diff
@@ -1,33 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.linecorp.linesdktest">
-    <!--
-     Chrome Custom Tab supports sdk version 16 or more.
-     This is needed because LINE SDK supports sdk version 15 or more.
-    -->
-    <uses-sdk tools:overrideLibrary="android.support.customtabs" />
+    xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
-        android:name=".LineSdkTestApplication"
-        android:allowBackup="false"
+        android:dataExtractionRules="@xml/data_extraction_rules"
```

Comment:
these two lines and corresponding files could be removed,

Reply:
Seems generated by Android Studio.
Addressed in [40825ee](https://github.com/line/line-sdk-android/pull/128/commits/40825ee5d14ce486576d9c37da07d6cf9f5a25f7).